### ISSUE TITLE
Command Palette: Action and command line mode in relation to char >

### ISSUE THREAD

**Issue by thernstig** [microsoft/terminal#7736]
# Description of the new feature/enhancement

In VS Code, pressing `ctrl+shift+p` opens the command palette with `>`.
In Windows terminal, pressing `ctrl+shift+p` opens the command palette without `>`.

But both mean the same thing i.e. if I enter some text I find commands to execute.

As I expect many users will use VS Code + Windows Terminal when developing on Windows, would it not be better to correlate these? The benefits are:

1. `>` means the same thing for both apps, which decreases cognitive load.
2. It is easier to press backspace to remove the `>` than it is to add the `>`.

I.e. I think the `>` should instead mean "action mode" whilst no `>` will mean command line mode, according to https://devblogs.microsoft.com/commandline/windows-terminal-preview-1-3-release/ (see under "Command palette").

I realize this might have been discussed when implementing this, but in that case and if this issue is closed, it might serve as a reference to other users being confused by this unfortunate discrepancy.

**Comment by zadjii-msft** [comment]
Oh my gosh you're right, we did end up doing this backwards from VsCode.

I even had it called out in the spec to do it the other way: [Cmd Palette Spec](https://github.com/microsoft/terminal/blob/master/doc/specs/%232046%20-%20Command%20Palette.md#switching-between-modes). That whole section is labeled with a bunch of "for discussion" tags, but I guess I never updated the spec to reflect the final decision-making point: https://github.com/microsoft/terminal/pull/5674#discussion_r427619170

> I'm now doubly on board with `>` being the Commandline mode key, because that's exactly what Powertoys Run picked for their "commandline mode" (whatever they're calling it)

We're definitely in a weird place here - we already shipped the CmdPal with `>` as the Commandline Mode prefix, so I'm not sure that we _could_ swap them now, though, I suppose all the 1.x releases are just previews of 2.0, and we still have _some_ runway for breaking changes...

**Comment by thernstig** [comment]
Glad I at least posted this then, and that it was an oversight. To make progress in the world of all that is coding, I do think a change is warranted and I am not sure many users would be angry if a change was made. The reason being that the default of pressing `ctrl+shift+p` would still by default end up in "action mode" just it it has the `>` in there, which is as I mentioned easy to delete with a backspace. It is much easier than adding `>` to go into "command line mode".

tldr; I think most users would find this an improvement (even if we do not take VS Code into account).

**Comment by zadjii-msft** [comment]
Summoning @DHowett, @cinnamon-msft. What do y'all think - do we follow PowerToys run, or VsCode?

**Comment by cinnamon-msft** [comment]
If we start with `>`, would it say "Enter a wt commandline to run" or would it still list the available command options?

**Comment by thernstig** [comment]
It would list the available commad options. Exactly like in VS Code.

**Comment by cinnamon-msft** [comment]
Then I think we should follow VS Code for this. We've followed them for the key binding, might as well keep it going. :)

**Comment by DHowett** [comment]
Meh, let's just flip it. It starts with `>` by default, and `>` is "action search mode".

FWIW: VS Code doesn't _have_ a commandline invocation mode, so there's no real parallel here.

How do other popular editors handle this? How about terminal emulators?

**Comment by thernstig** [comment]
@DHowett 

> FWIW: VS Code doesn't have a commandline invocation mode, so there's no real parallel here.

That is true, but for what it is worth it has another mode (file mode) when the `>` is removed.

**Comment by zadjii-msft** [comment]
Thanks for the feedback y'all. I'll throw this on my 1.5 plate, so we don't ship the CmdPal with the original behavior for _too_ long.

I'm not sure there's any other precedent for a command palette in other terminal emulators, so that's not really a factor here. 

Sublime Text's "action mode" has no prefix, but also so does the "navigate to file" mode. They're just on separate keybinds. "Go to reference" (e.g. function name) is on the prefix `@`, from the "navigate to file" mode, and "go to line" is on `:`. Similar to VS Code, but I think the key difference is the ability to get to "action mode" with `>` in VS.

**Comment by zadjii-msft** [comment]
Oof, this is a little trickier than just "flip the logic around" - we're going to have to fake out the placeholder text for action mode now, because it will have a literal `>` in the input, so XAML won't display the `TextBox`'s `PlaceholderText`. 

Annoying, but fixable.
